util/rotationRepresentationSize.m

```diff
@@ -0,0 +1,15 @@
+function ret = rotationRepresentationSize(rotation_type)
```

Comment:
the notation of the rotation_type only makes sense in the context of our RBM classes.  so I think this should as a static method of RBM, not in the util folder